packages/block-library/src/icon/icons/index.js

```diff
@@ -0,0 +1,470 @@
+		title: __( 'Comment Author Avatar' ),
+		icon: commentAuthorAvatar,
+		categories: [ 'blocks' ],
+	},
+	{
+		name: 'desktop',
+		title: __( 'Desktop' ),
+		icon: desktop,
+		categories: [ 'design' ],
+	},
+	{
+		name: 'download',
+		title: __( 'Download' ),
+		icon: download,
+		categories: [ 'media' ],
+	},
+	{
+		name: 'edit',
+		title: __( 'Edit' ),
+		icon: edit,
+	},
+	{
+		name: 'external',
+		title: __( 'External' ),
+		icon: external,
+	},
+	{
+		name: 'file',
+		title: __( 'File' ),
+		icon: file,
+	},
+	{
+		name: 'gallery',
+		title: __( 'Gallery' ),
+		icon: gallery,
+		categories: [ 'blocks' ],
+	},
+	{
+		name: 'help',
+		title: __( 'Help' ),
+		icon: help,
+	},
+	{
+		name: 'helpFilled',
+		title: __( 'Help - Filled' ),
+		icon: helpFilled,
+	},
+	{
+		name: 'image',
+		title: __( 'Image' ),
+		icon: image,
+		categories: [ 'blocks' ],
+	},
+	{
+		name: 'info',
+		title: __( 'Info' ),
+		icon: info,
+	},
+	{
+		name: 'link',
+		title: __( 'Link' ),
+		icon: link,
+	},
+	{
+		name: 'lockOutline',
+		title: __( 'Lock - Outline' ),
+		icon: lockOutline,
+	},
+	{
+		name: 'media',
+		title: __( 'Media' ),
+		icon: media,
+		categories: [ 'blocks' ],
+	},
+	{
+		name: 'megaphone',
+		title: __( 'Megaphone' ),
+		icon: megaphone,
+	},
+	{
+		name: 'mobile',
+		title: __( 'Mobile' ),
+		icon: mobile,
+		categories: [ 'design' ],
+	},
+	{
+		name: 'page',
+		title: __( 'Page' ),
+		icon: page,
+		categories: [ 'blocks' ],
+	},
+	{
+		name: 'people',
+		title: __( 'People' ),
+		icon: people,
+	},
+	{
+		name: 'plugins',
+		title: __( 'Plugins' ),
+		icon: plugins,
+	},
+	{
+		name: 'plus',
+		title: __( 'Plus' ),
+		icon: plus,
+	},
+	{
+		name: 'quote',
+		title: __( 'Quote' ),
+		icon: quote,
+		categories: [ 'blocks' ],
+	},
+	{
+		name: 'redo',
+		title: __( 'Redo' ),
+		icon: redo,
+	},
+	{
+		name: 'resizeCornerNE',
+		title: __( 'Resize Corner - Northeast' ),
+		icon: resizeCornerNE,
+	},
+	{
+		name: 'rss',
+		title: __( 'RSS' ),
+		icon: rss,
+		categories: [ 'blocks' ],
+	},
+	{
+		name: 'search',
+		title: __( 'Search' ),
+		icon: search,
+		categories: [ 'blocks' ],
+	},
+	{
+		name: 'seen',
+		title: __( 'Seen' ),
+		icon: seen,
+	},
+	{
+		name: 'settings',
+		title: __( 'Settings' ),
+		icon: settings,
+	},
+	{
+		name: 'starEmpty',
+		title: __( 'Star - Empty' ),
+		icon: starEmpty,
+	},
+	{
+		name: 'starFilled',
+		title: __( 'Star - Filled' ),
+		icon: starFilled,
+	},
+	{
+		name: 'store',
+		title: __( 'Store' ),
+		icon: store,
+	},
+	{
+		name: 'trash',
+		title: __( 'Trash' ),
+		icon: trash,
+	},
+	{
+		name: 'trendingDown',
+		title: __( 'Trending Down' ),
+		icon: trendingDown,
+	},
+	{
+		name: 'trendingUp',
+		title: __( 'Trending Up' ),
+		icon: trendingUp,
+	},
+	{
+		name: 'undo',
+		title: __( 'Undo' ),
+		icon: undo,
+	},
+	{
+		name: 'video',
+		title: __( 'Video' ),
+		icon: video,
+		categories: [ 'blocks' ],
+	},
+	{
+		name: 'warning',
+		title: __( 'Warning' ),
+		icon: warning,
```

Comment:
```suggestion
		name: 'cautionFilled',
		title: __( 'Warning' ),
		icon: cautionFilled,
```

The `warning` icon is deprecated, and we need to use `cautionFilled` instead.

I think #71878 removed [alias imports](https://github.com/WordPress/gutenberg/blob/24b48886eec82287eef3e7b31f2122933f1ad038/packages/icons/src/index.ts#L41-L45), which will cause build errors.

cc @youknowriad @mcsf